src/winetricks

```diff
@@ -6682,7 +6685,9 @@ load_dotnet45()
     w_call remove_mono
 
     # See https://appdb.winehq.org/objectManager.php?sClass=version&iId=25478 for Focht's recipe
-    w_call dotnet35
+
+    # Seems unneeded in wine-2.0
```

Comment:
From my understanding based on bug reports, etc. the issue is that the .NET installers, while using a Wine-compatible install method for that version, it uses MSUs to install the previous versions. In Windows, this means that installing .NET 4.5 installs .NET 4.5, 4.0, 3.5, 3.0, 2.0, and 1.1.  In Wine non-staging (or staging < 1.8), the MSU installer is just a stub that fakes a successful install, so the .NET installation seems to go fine, but is incomplete.  The installer file should have its contents extracted and the MSUs examined to make sure no files are needed from .NET 3.5 or below.